main_routers/system_router.py

```
@@ -1876,8 +1877,25 @@ async def proxy_meme_image(url: str):
+        # 已知 SSL 证书有问题的 CDN 域名（如七牛 CDN hostname mismatch），
+        # 对这些域名首次请求即使用宽松 SSL，避免白白浪费一次超时。
+        _SSL_RELAXED_HOSTS = {'qn.doutub.com'}
+        need_relaxed_ssl = hostname in _SSL_RELAXED_HOSTS
+
+        def _make_client(relaxed: bool = False) -> httpx.AsyncClient:
+            if relaxed:
+                ctx = ssl.create_default_context()
+                try:
+                    ctx.set_ciphers('DEFAULT@SECLEVEL=1')
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)